Dockerfile

```diff
@@ -60,17 +68,20 @@ ENV MENDER_CLIENT_VERSION=$mender_client_version
 
 # NOTE: we are assuming generic ARM board here, needs to be extended later
 
-ENV PATH "$PATH:/usr/local/go/bin"
+ENV PATH "$PATH:/usr/local/go/bin:/armv6-eabihf--glibc--stable-2018.11-1/bin"
 ENV GOPATH "/root/go"
 
 RUN go get -d github.com/mendersoftware/mender
 WORKDIR $GOPATH/src/github.com/mendersoftware/mender
 RUN git checkout $MENDER_CLIENT_VERSION
 
+ENV CC "arm-buildroot-linux-gnueabihf-gcc"
```

Comment:
Good.
But we need to be consistent now and use only one toolchain. So I think we need to:
- Remove gcc-arm-linux-gnueabihf from the apt-get packages to install
- Modify the --host tag in the ./configure command for liblzma
- Delete the extra CC param in the mender make build call

Reply:
Amendment on its way. Making PR during Sprint review is not recommended.

Reply:
Hehe. No problem. This part is good now